py/server/deephaven/parquet.py

```diff
@@ -151,9 +191,16 @@ def write(table: Table, path: str, col_definitions: List[Column] = None,
         raise DHError(e, "failed to write to parquet data.") from e
 
 
-def batch_write(tables: List[Table], paths: List[str], col_definitions: List[Column],
-                col_instructions: List[ColumnInstruction] = None, compression_codec_name: str = None,
-                max_dictionary_keys: int = None, grouping_cols: List[str] = None):
+def batch_write(
+    tables: List[Table],
+    paths: List[str],
+    col_definitions: List[Column],
+    col_instructions: List[ColumnInstruction] = None,
+    compression_codec_name: str = None,
+    max_dictionary_keys: int = None,
+    target_page_size: int = None,
```

Comment:
It is fine to have a default value of None, but maybe we should document if unset, what the target_page_size is.